test/unit/index.test.ts

```diff
@@ -227,10 +230,125 @@ describe('index', (): void => {
         // Assert
         expect(response.statusCode).toBe(400)
         expect(response.result).toStrictEqual(expected)
-        expect(mockForwardAuthorizationPost).not.toHaveBeenCalled()
+        expect(mockForwardAuthorizationRequest).not.toHaveBeenCalled()
       })
     })
 
+    describe('PUT /thirdpartyRequests/transactions/{ID}/authorizations', (): void => {
+      beforeAll((): void => {
+        mockLoggerPush.mockReturnValue(null)
+        mockLoggerError.mockReturnValue(null)
+      })
+
+      beforeEach((): void => {
+        jest.clearAllMocks()
+      })
+
+      it('PUT', async (): Promise<void> => {
+        mockForwardAuthorizationRequest.mockResolvedValueOnce()
+        const request = {
+          method: 'PUT',
+          url: '/thirdpartyRequests/transactions/7d34f91d-d078-4077-8263-2c047876fcf6/authorizations',
+          headers: {
+            accept: 'application/json',
+            date: (new Date()).toISOString(),
+            'fspiop-source': 'dfspA',
+            'fspiop-destination': 'dfspA'
+          },
+          payload: {
+            challenge: '12345',
+            value: '12345',
+            consentId: '8e34f91d-d078-4077-8263-2c047876fcf6',
+            sourceAccountId: 'dfspa.alice.1234',
+            status: 'VERIFIED'
+          }
+        }
+        const expected = [
+          '/thirdpartyRequests/transactions/{{ID}}/authorizations',
+          'THIRDPARTY_CALLBACK_URL_TRANSACTION_REQUEST_AUTHORIZATIONS_PUT',
+          expect.objectContaining(request.headers),
+          'PUT',
+          '7d34f91d-d078-4077-8263-2c047876fcf6',
+          request.payload,
+          expect.any(Object)
+        ]
+
+        // Act
+        const response = await server.inject(request)
+
+        // Assert
+        expect(response.statusCode).toBe(202)
+        expect(response.result).toBeNull()
+        expect(mockForwardAuthorizationRequest).toHaveBeenCalledWith(...expected)
+      })
+
+      it('responds with a 400 when status !== VERIFIED', async (): Promise<void> => {
```

Comment:
See above comments on the `api-template.yaml`. I think this is right, but we need to think forward a little bit about error handling.

Reply:
Ok